README.md

```diff
@@ -155,6 +156,8 @@ Also ensure the `command` field in `docker-compose.yml` is modified for `CUDA_VI
 
 MongoDB will work "out-of-the-box", however, if you need to do something specific, please refer the following section.
 
+Note: If initializing from a checkpoint, it may take time to perform the initial restore.
+
```

Comment:
at line 165, it is not accurate.
We may need review the whole README and update everywhere which is for old version and out of date.